spec/features/feedex_spec.rb

```diff
@@ -103,4 +104,164 @@
+    org_summary_request = stub_support_api_anonymous_feedback_organisation_summary('cabinet-office', 'last_7_days', {
+      "title" => "Cabinet Office",
+      "slug" => 'cabinet-office',
+      "anonymous_feedback_counts" => [
+        { path: '/vat-rates', last_7_days: 5, last_30_days: 10, last_90_days: 20 },
+        { path: '/guidance/doing-fun-things' },
+        { path: '/guidance/not-doing-bad-things' },
+      ],
+    })
+
+    org_feedback_request = stub_support_api_anonymous_feedback(
+      { organisation_slug: "cabinet-office" },
+      {
+        "current_page" => 1,
+        "pages" => 1,
+        "page_size" => 3,
+        "results" => [
+          {
+            type: "problem-report",
+            path: "/vat-rates",
+            url: "http://www.dev.gov.uk/vat-rates",
+            created_at: DateTime.parse("2013-03-01"),
+            what_doing: "looking at 3rd paragraph",
+            what_wrong: "typo in 2rd word",
+            referrer: "https://www.gov.uk/",
+          },
+          {
+            type: "problem-report",
+            path: "/guidance/doing-fun-things",
+            url: "http://www.dev.gov.uk/guidance/doing-fun-things",
+            created_at: DateTime.parse("2013-02-01"),
+            what_doing: "finding out about fun",
+            what_wrong: "no mention of petting a dog",
+            referrer: "https://www.gov.uk/having-fun",
+          },
+          {
+            type: "problem-report",
+            path: "/guidance/doing-bad-things",
+            url: "http://www.dev.gov.uk/guidance/not-doing-bad-things",
+            created_at: DateTime.parse("2013-02-01"),
+            what_doing: "looking at what you consider bad",
+            what_wrong: "so many typos, whoever wrote this must have been angry",
+            referrer: "https://www.gov.uk/having-fun",
+          },
+        ]
+      }
+    )
+
+    org_and_path_feedback_request = stub_support_api_anonymous_feedback(
+      { organisation_slug: "cabinet-office", path_prefix: '/guidance' },
+      {
+        "current_page" => 1,
+        "pages" => 1,
+        "page_size" => 2,
+        "results" => [
+          {
+            type: "problem-report",
+            path: "/guidance/doing-fun-things",
+            url: "http://www.dev.gov.uk/guidance/doing-fun-things",
+            created_at: DateTime.parse("2013-02-01"),
+            what_doing: "finding out about fun",
+            what_wrong: "no mention of petting a dog",
+            referrer: "https://www.gov.uk/having-fun",
+          },
+          {
+            type: "problem-report",
+            path: "/guidance/doing-bad-things",
+            url: "http://www.dev.gov.uk/guidance/doing-bad-things",
+            created_at: DateTime.parse("2013-02-01"),
+            what_doing: "looking at what you consider bad",
+            what_wrong: "so many typos, whoever wrote this must have been angry",
+            referrer: "https://www.gov.uk/having-fun",
+          },
+        ]
+      }
```

Comment:
Can't this hash re-use the one used here: https://github.com/alphagov/support/pull/318/commits/01451a9898ceb7fc8d2fea9a9467c39815139ae1#diff-2831018522e781bb592490f9d762a4a6R122

As far as I can tell the thing that differs between these requests is the params in the first hash, so the second hash could probably be defined once and then re-used everywhere.